### PR TITLE
Set permissions for GitHub actions

### PR DESCRIPTION
**Pull request by naveensrinivasan**
- Included permissions for the action. https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Comment by BenTheElder** [comment]
Do pull_request workflows have more than read by default? ⚠️

File: .github/workflows/cgroup2.yaml [hunk 1]
```diff
@@ -29,7 +32,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Boot Fedora
-        run: |
+        un: |
```
Comment (BenTheElder) [inline .github/workflows/cgroup2.yaml:35]: this change and the r at the end of the file appear to be errant

**Comment by stmcginnis** [comment]
> Do pull_request workflows have more than read by default?

Looks like it is a repo setting. https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-default-github_token-permissions

Might be good to have someone with permissions to that area to check the settings.

**Comment by BenTheElder** [comment]
I just changed it 

A6A6BED0-C73C-4337-8288-A92F4B44BB10

File: .github/workflows/cgroup2.yaml (hunk 1 above)
Comment (naveensrinivasan) [reply .github/workflows/cgroup2.yaml]: Thanks, took care of it.

**Comment by aojea** [comment]
should it be?
```
permissions: read-all
```

**Comment by BenTheElder** [comment]
@aojea no, we shouldn't need to grant access to other permissions and from the docs

> If you specify the access for any of these scopes, all of those that are not specified are set to none.

**Pull request merged**